service/hook/bitbucketv2/bitbucketv2.go

```diff
@@ -175,6 +180,7 @@ func transformPushEvent(pushEvent PushEventModel) hookCommon.TransformResultMode
 					CommitMessage:     aNewItm.Target.CommitMessage,
 					BaseRepositoryURL: pushEvent.RepositoryInfo.getRepositoryURL(),
 				},
+				// todo: do we need website-provider/user here?
```

Comment:
Yes. There are multiple trigger actions like creating a tag, push or pull-request. We need contributor for all type which ended up in triggering a new Bitrise build.

Reply:
Updated